packages/terra-clinical-result-view/src/ResultView.scss

```diff
@@ -0,0 +1,114 @@
+@import './mixins';
+
+:local {
+  .result-view {
+    color: var(--result-view-background-color, #595f62);
+    font-size: 20px;
```

Comment:
Do we want font-size and line-height themeable here and in the other css classes?